hyperapp.d.ts

```diff
@@ -0,0 +1,149 @@
+declare module "hyperapp" {
```

Comment:
Both are similar but different. declaring module are more ES6/CommonJS where declaring namespace is more like exposing a global object. 
This is why if you export and object you will need to export the namespace too.

````tsx
export = HyperApp

declare namespace HyperApp {
  ...
}